### PR TITLE
ACMS-1862: Tests deprecation fixes with D10.1

### PR DESCRIPTION
**Pull request by apathak18**
**Motivation**

Fixes ACMS-1862

### CODE REVIEW

**Review by chandan-singh7929 — APPROVED** [review]
LGTM

**Review by rajeshreeputra — APPROVED** [review]
Changes looks good to me.

File: tests/src/ExistingSiteJavascript/CohesionTestBase.php [hunk 1]
```diff
@@ -34,6 +34,9 @@ protected function setUp(): void {
    *   A wrapper object for interacting with the layout canvas.
    */
   protected function getLayoutCanvas(): LayoutCanvas {
+    /** @var \Drupal\FunctionalJavascriptTests\JSWebAssert $assertSession */
+    $assertSession = $this->assertSession();
+    $assertSession->waitForElementVisible('css', '.ssa-layout-canvas');
```
Comment (vishalkhode1) [inline tests/src/ExistingSiteJavascript/CohesionTestBase.php:39]: @apathak18 I'm reverting this change, as the same is happening at line #40. If you still think, this is needed, let's discuss and create a separate PR.

Comment (apathak18) [reply tests/src/ExistingSiteJavascript/CohesionTestBase.php]: Yes this needs to be revert.

**Pull request merged**